catalyst_voices/apps/voices/lib/widgets/modals/details/voices_details_dialog.dart

```diff
@@ -5,24 +5,30 @@ import 'package:flutter/material.dart';
 class VoicesDetailsDialog extends StatelessWidget {
   final Widget header;
   final Widget body;
+  final BoxConstraints? constraints;
+  final Color? backgroundColor;
 
   const VoicesDetailsDialog({
     super.key,
     required this.header,
     required this.body,
+    this.constraints,
+    this.backgroundColor,
   });
 
   @override
   Widget build(BuildContext context) {
     return VoicesSinglePaneDialog(
       showClose: false,
       showBorder: true,
-      constraints: const BoxConstraints(
-        minWidth: 600,
-        maxWidth: 900,
-        minHeight: double.infinity,
-      ),
-      backgroundColor: Theme.of(context).colors.elevationsOnSurfaceNeutralLv0,
+      constraints: constraints ??
+          const BoxConstraints(
+            minWidth: 600,
+            maxWidth: 900,
+            minHeight: double.infinity,
+          ),
```

Comment:
please make `constraints` not nullable and use 
```dart
const BoxConstraints(
            minWidth: 600,
            maxWidth: 900,
            minHeight: double.infinity,
          )
```

as default value in constructor